cluster/gce/manifests/etcd.manifest

```diff
@@ -21,6 +21,7 @@
         "cpu": {{ cpulimit }}
       }
     },
+    "comment": "If etcd name defined by --name is modified, please make sure ETCD_NAME in migrate-if-needed.sh is changed to the same: https://github.com/kubernetes/kubernetes/blob/v1.19.0-beta.2/cluster/images/etcd/migrate-if-needed.sh#L49.",
```

Comment:
Hmm - that is strange. You're adding a field to pod.spec.container.

That field obviously doesn't exist. It works because kube-apiserver will drop this field, but that's definitely NOT what I would expect...

Reply:
hmm, this is json file, I don't think comment is supported.

Reply:
Shall we add a _comment field or just not add the comment?